lib/presentation/pages/tip_information_page.dart

```diff
@@ -49,7 +49,12 @@ class TipInformationPage extends StatelessWidget {
                         const SeparatorAtom(
                           variant: SeparatorVariant.relatedElements,
                         ),
-                        TextAtom(tip.text),
+                        Flexible(
+                          child: TextAtom(
+                            tip.text,
+                            overflow: TextOverflow.clip,
```

Comment:
Can you explain why clip is needed?

Reply:
Clip specifies the overflow behaviour of the text, ensuring that the text always clips. It would work without that as well but I added the property just in case